project_multifamily_beta/pat.json

```diff
@@ -494,7 +494,11 @@
       "seed": "EmptySeedModel.osm",
       "workflow_index": 1,
       "options": [],
+<<<<<<< HEAD
       "instanceId": 0.5261009231493268,
+=======
+      "instanceId": 0.4202025159376823,
+>>>>>>> master
```

Comment:
It looks like you committed a bunch of conflicts 👀

Reply:
Thanks, these have been corrected.